plugins/modules/nd_backup_schedule.py

```diff
@@ -0,0 +1,210 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright: (c) 2025, Sabari Jaganathan (@sajagana) <sajagana@cisco.com>
+# GNU General Public License v3.0+ (see LICENSE or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+
+__metaclass__ = type
+
+ANSIBLE_METADATA = {"metadata_version": "1.1", "status": ["preview"], "supported_by": "community"}
+
+DOCUMENTATION = r"""
+---
+module: nd_backup_schedule
+version_added: "0.5.0"
+short_description: Manages backup schedule on Cisco Nexus Dashboard.
+description:
+- Manage backup schedule on Cisco Nexus Dashboard.
+- This module is only supported on ND v4.1 and later.
+author:
+- Sabari Jaganathan (@sajagana)
+options:
+  name:
+    description:
+    - The name of the backup schedule.
+    type: str
+  encryption_key:
+    description:
+    - The encryption_key for a backup file.
+    type: str
+  remote_location:
+    description:
+    - The name of the remote storage location.
+    type: str
+  frequency:
+    description:
+    - The frequency at which remote backups are scheduled to occur at specified intervals on selected days.
+    type: int
+  scheduler_date:
+    description:
+    - The start date for the backup schedule in the format O(scheduler_date="YYYY-MM-DD").
+    type: str
+    aliases: [ scheduler_start_date, start_date, date ]
+  scheduler_time:
+    description:
+    - The start time for the backup schedule in the format O(scheduler_date="HH-MM-SS").
+    type: str
+    aliases: [ scheduler_start_time, start_time, time ]
+  backup_type:
+    description:
+    - The O(backup_type=config_only) option creates a snapshot that specifically captures the configuration settings of the Nexus Dashboard.
```

Comment:
Can we add a description for backup_type?

- This parameter specifies the kind of snapshot created for the Nexus Dashboard.

Reply:
Done!